docs/maker.md

```diff
@@ -0,0 +1,141 @@
+# Maker CLI Documentation
+
+The maker CLI is a command-line interface for interacting with the maker server in the Coinswap protocol. It allows you to manage your maker wallet, view balances, and perform various operations.
+
+## Overview
+
+The maker CLI provides functionality to:
+- Monitor wallet balances
+- View UTXOs
+- Generate new addresses
+- Send funds to external addresses
+- Get network information
+
+## Installation
+
+The maker CLI is part of the Coinswap project. To build from source:
+
+```bash:docs/maker-docs.md
+git clone https://github.com/citadel-tech/coinswap
+cd coinswap
+cargo build --release
+```
+
+## Commands
+
+### Basic Commands
+
+1. **Ping**
+   - Tests connectivity to the maker server
+   ```bash
+   maker-cli ping
+   ```
+
+2. **View UTXOs**
+   - List seed UTXOs: `maker-cli seed-utxo`
+   - List swap coin UTXOs: `maker-cli swap-utxo`
+   - List contract UTXOs: `maker-cli contract-utxo`
+   - List fidelity UTXOs: `maker-cli fidelity-utxo`
+
+3. **Check Balances**
+   - Seed balance: `maker-cli seed-balance`
+   - Swap coin balance: `maker-cli swap-balance`
+   - Contract balance: `maker-cli contract-balance`
+   - Fidelity balance: `maker-cli fidelity-balance`
+
+4. **Address Management**
+   - Generate new address: `maker-cli new-address`
+
+5. **Send Funds**
+   ```bash
+   maker-cli send-to-address --address <BITCOIN_ADDRESS> --amount <SATS> --fee <SATS>
+   ```
+
+6. **Network Information**
+   - Get Tor address: `maker-cli get-tor-address`
+   - Get data directory: `maker-cli get-data-dir`
+
+## Configuration
+
+The maker CLI uses a default data directory at `~/.coinswap/maker`. This directory contains:
+- Wallet data
+- Debug logs
+- Configuration files
+
+## Fidelity Bonds
+
+The maker CLI supports fidelity bond operations. These bonds are used as a Sybil resistance mechanism. The value of fidelity bonds is calculated based on:
+
+- Time-locked amount
+- Lock period
+- Current time
+- Interest rate (1.5% default)
+- Bond value exponent (1.3 default)
+
+The bond value calculation uses a compound interest formula to incentivize longer lock periods and higher amounts.
```

Comment:
Instead of explaining what fidelity bond is, just mention the maker uses fidelity bonds as "costly identity". These bonds are created automatically on startup. If the maker doesn't have enough initial balance, a log will be printed asking for the funds to a specific address.

Then paste the printout of the log, where it asks for fidelity amounts. 

Add a link to the doc repo fidelity.md file for further reading reference on fidelity bonds.